src/prec.rs

```diff
@@ -9,20 +53,128 @@ pub const F64_PREC: f64 = 0.00000000000000011102230246251565;
 /// Default accuracy for `f64`, equivalent to `0.0 * F64_PREC`
```

Comment:
This has not been changed in this PR, but I just realized that this is incorrect. would you mind including a fix for this here?